lib/modules/orangered.js

```diff
@@ -171,9 +166,32 @@ const setupFaviconBadge = _.once(() => {
 	return favicon;
 });
 
-function setupFloatingButtons() {
-	if (!module.options.showFloatingEnvelope.value) return;
-	if ($floatingInboxCount) return;
+const orangeredElements = () => ({
```

Comment:
You could also wrap this in `_.once` if you want so you aren't selecting `#modmail` every time